frontend/package.json

```diff
@@ -5,9 +5,11 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "dev:wails": "VITE_NWC_APP_TYPE=WAILS vite",
+    "dev:wails": "yarn prepare:wails && VITE_NWC_APP_TYPE=WAILS vite",
```

Comment:
VITE_NWC_APP_TYPE should not be needed now, right?

Should we delete the original request.ts file and add it to .gitignore?

Reply:
if you think that's a doable way, we can go forward with that.